style/ember/README.md

```diff
@@ -2,6 +2,9 @@ Ember
 =====
 
 * Don't put a space between the opening handlebars braces and the variable.
+* Alias local versions of functions from `Ember` and `DS`. These will eventually
+  be moved to JavaScript modules allowing you to only import the functions you
+  require ([sample][local-Ember-DS])
```

Comment:
I'm in favor of this, but does the guideline need to embed the motivation behind it?